lib/stream.js

```diff
@@ -65,10 +65,7 @@ function createWebSocketStream(ws, options) {
   const duplex = new Duplex({
     ...options,
     autoDestroy: false,
-    emitClose: false,
-    objectMode: false,
```

Comment:
Should users be allowed to change this? If so why?

Reply:
Changed it as you requested in previous comment. In practice I'd like to be able to set `readableObjectMode` and `encoding`.